integration/nwo/fabric/opts/opts.go

```diff
@@ -104,6 +104,20 @@ func (o *Options) SetAnonymousIdentity(v bool) {
 	o.Mapping["AnonymousIdentity"] = v
 }
 
+// HSMDefaultIdentity return true if the default identity is an HSM identity
+func (o *Options) HSMDefaultIdentity() bool {
+	res := o.Mapping["HSMDefaultIdentity"]
+	if res == nil {
+		return false
+	}
+	return res.(bool)
+}
```

Comment:
This does not feel intuitive to me. What is our API here?

In my view we have X509 Identities and Idemix Identities. In case of X509, they can be provided via HSM or some keystore/file.
We also use the concept of `DefaultIdentity`, which defines whatever identity should be used as default identity. 
If you want to know if an identity is provided by HSM or not, or in particular, if you want to know if the default identity is provided by an HSM, what question would we ask? `IsDefaultIdentityProvidedByHSM() bool`, or `IsIdentityProvidedByHSM(identity string) bool`, or `IsHSMIdentity(identity string) bool` ...